### ISSUE TITLE
Question: Keyboard shortcuts?

### ISSUE THREAD

**Issue by javadba** [tabulapdf/tabula#1081]
Are there any keyboard shortcuts for this app? If so is there a document/list for them?

**Comment by jeremybmerrill** [comment]
Nope, though I suppose we might consider adding them if we received a pull request. What sorts of actions would you want keyboard shortcuts for?
